src/core/text/TextStyle.js

```diff
@@ -435,22 +435,43 @@ export default class TextStyle
  * @param {number|number[]} color
  * @return {string} The color as a string.
  */
-function getColor(color)
+function getSingleColor(color)
 {
     if (typeof color === 'number')
     {
         return hex2string(color);
     }
-    else if (Array.isArray(color))
+    else if ( typeof color === 'string' )
     {
-        for (let i = 0; i < color.length; ++i)
+        if ( color.includes('0x') )
```

Comment:
The ES6 API way to do this would be color.startsWith('0x'), but we aren't there yet.